leaf-ui/js/object/evoObjects.js

```diff
@@ -25,19 +25,92 @@ class IntentionColorVis {
 
 /**
  * Colors the nodes based on the different EVO types 
+ * This order is created intentionally for the frontend. Please DO NOT change the order
  */
 class EVO {
+
     static colorVisDict = {
+        "0000": "#D3D3D3", // None
+        "0011": "#003fff", // Satisfied
+        "0010": "#8FB8DE", // Partially satisfied
+        "0100": "#fbaca8", // Partially denied
+        "0110": "#9400D3", // Conflict 
+        "0111": "#5946b2",
```

Comment:
I think this is a conflict right? Should we add a comment for it, and in lines 40 and 41

Reply:
great idea! now fully documented